.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Make sure these boxes are checked before submitting the issue - Thank you for the contribution.
```

Comment:
Rather than making sure to check boxes, just ask to provide the info and provide a space to do that.